src/helpers/activitypub/actor.unit.test.ts

```diff
@@ -226,3 +235,123 @@ describe('isHandle', () => {
         expect(isHandle('@foo@@example.com')).toBe(false);
     });
 });
+
+describe('updateSiteActor', () => {
+    function mockApContext(db: KvStore, globaldb: KvStore) {
+        return {
+            data: {
+                db,
+                globaldb,
+                logger: console as unknown as Logger,
+            },
+            getActor: vi.fn().mockResolvedValue({}),
+            getInboxUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/inbox')),
+            getOutboxUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/outbox')),
+            getLikedUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/liked')),
+            getFollowingUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/following')),
+            getActorUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/user/1')),
+            getActorKeyPairs: vi.fn().mockReturnValue([
+                {
+                    cryptographicKey: 'abc123',
+                },
+            ]),
+            getObjectUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com')),
+            getFollowersUri: vi
+                .fn()
+                .mockReturnValue(new URL('https://example.com/followers')),
+            sendActivity: vi.fn(),
+        } as unknown as RequestContext<ContextData>;
+    }
+
+    it('should return false if the site settings have not changed', async () => {
+        const db = {
+            get: vi.fn().mockResolvedValue({
+                id: 'https://example.com/user/1',
+                name: 'Site Title',
+                summary: 'Site Description',
+                preferredUsername: 'index',
+                icon: 'https://example.com/icon.png',
+                inbox: 'https://example.com/inbox',
+                outbox: 'https://example.com/outbox',
+                following: 'https://example.com/following',
+                followers: 'https://example.com/followers',
+                liked: 'https://example.com/liked',
+                url: 'https://example.com',
+            }),
+            set: vi.fn(),
+        } as unknown as KvStore;
+
+        const globaldb = {
+            get: vi.fn().mockResolvedValue(null),
+            set: vi.fn(),
+        } as unknown as KvStore;
+
+        const getSiteSettings = vi.fn().mockResolvedValue({
+            site: {
+                description: 'Site Description',
+                title: 'Site Title',
+                icon: 'https://example.com/icon.png',
+            },
+        });
+
+        const host = 'example.com';
+
+        const apCtx = mockApContext(db, globaldb);
+
+        const result = await updateSiteActor(apCtx, getSiteSettings, host);
+
+        assert(result === false);
```

Comment:
```suggestion
        expect(result).toBe(false);
```

For consistency with other tests in this file